### PR TITLE
Section Cycles, task 15, 16

### PR DESCRIPTION
**Pull request by veronaten**

### CODE REVIEW

File: scripts/cycles.js [hunk 1]
```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
     }
   }
 }
+
+let n = 1000;
```
Comment (magicsd) [inline scripts/cycles.js]: Why is this variable out of scope of the function it used in?

File: scripts/cycles.js [hunk 2]
```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
...
+  }
+
+  for (num = 1; num <= iteration; num++) {
+    n = n / 2;
+    if (n > 50) {
+      alert(`${n} , not correct, try increasing the number !!!`);
+    } else if ( n > 30 && n < 50) {
+      alert(`${n} , yes, tha's 5 times!!!`);
+    } else if (n < 30) {
+      alert(`${n}  , not correct, try reducing the number`);
```
Comment (magicsd) [inline scripts/cycles.js]:
```suggestion
      alert(`${n} is not correct, try to reduce the number`);
```

File: scripts/cycles.js [hunk 3]
```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
...
+
+    return;
+  }
+
+  for (num = 1; num <= iteration; num++) {
+    n = n / 2;
+    if (n > 50) {
+      alert(`${n} , not correct, try increasing the number !!!`);
+    } else if ( n > 30 && n < 50) {
+      alert(`${n} , yes, tha's 5 times!!!`);
```
Comment (magicsd) [inline scripts/cycles.js]:
```suggestion
      alert(`${n}: yes, that's 5 times!!!`);
```

File: scripts/cycles.js [hunk 4]
```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
...
+  if (Number.isNaN(iteration)) {
+    alert('Enter number, please.');
+
+    return;
+  }
+
+  for (num = 1; num <= iteration; num++) {
+    n = n / 2;
+    if (n > 50) {
+      alert(`${n} , not correct, try increasing the number !!!`);
```
Comment (magicsd) [inline scripts/cycles.js]:
```suggestion
      alert(`${n}: not correct, try to increase the number!!!`);
```

File: scripts/cycles.js [hunk 5]
```diff
@@ -17,3 +17,27 @@ function chooseWeekDays() {
     }
   }
 }
+
+let n = 1000;
+
+function countNumber() {
+  const iteration = Number(prompt('How many times do you need to divide 1000 by 2 until the result becomes < 50 ? Enter the number', '1') || 0);
```
Comment (magicsd) [inline scripts/cycles.js]:
```suggestion
  const iterationCount = Number(prompt('How many iterations do you need to divide 1000 by 2 until the result becomes < 50? Enter the number:', '1') || 0);
```

File: scripts/cycles.js (hunk 1 above)
Comment (veronaten) [reply scripts/cycles.js]: This is my mistake. I just wrote down my thoughts in order.

File: scripts/cycles.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  for (num = 1; num <= iterationCount; num++) {
+    n = n / 2;
+    if (n > 50) {
+      alert(`${n} is not correct, try to increase the number!!!`);
+    } else if ( n > 30 && n < 50) {
+      alert(`${n}: yes, that's 5 times!!!`);
+    } else if (n < 30) {
+      alert(`${n} is not correct, try to reduce the number`);
+      break;
+    }
```
Comment (magicsd) [inline scripts/cycles.js:41]: `break`???
```suggestion
    } else {
      alert(`${n} is not correct, try to reduce the number`);
    }  
```

Comment (veronaten) [reply scripts/cycles.js:41]: I want to break the cycle if the user enters a large number, such as 100.

**Pull request merged**